lib/loaders/loaders_with_authentication/tracked_entity.js

```diff
@@ -6,9 +6,9 @@ import { map, find, extend } from "lodash"
  * to accept an `ids` param for this to work.
  *
  * @param {*} dataLoader a configured (with path and default params) data loader that will make the actual API request
- * @param {*} paramKey the key that’s to be used for the list of IDs param that’s sent with the API request
- * @param {*} trackingKey the key that’s to be used for this entity, e.g. `is_followed` or `is_saved`.
- * @param {*} entityKeyPath an optional path to a nested entity
+ * @param {string} paramKey the key that’s to be used for the list of IDs param that’s sent with the API request
+ * @param {string} trackingKey the key that’s to be used for this entity, e.g. `is_followed` or `is_saved`.
+ * @param {string} [entityKeyPath] an optional path to a nested entity
```

Comment:
When adding `// @ts-check` to rules that use the tracked_entity function, I got errors around the number of params. Using `[paramName]` tells TS the param is optional.

Reply:
👍